compiler/resolution/wrappers.cpp

```diff
@@ -216,8 +219,11 @@ buildDefaultWrapper(FnSymbol* fn,
       ArgSymbol* wrapper_formal = copyFormalForWrapper(formal);
       if (fn->_this == formal)
         wrapper->_this = wrapper_formal;
-      if (formal->hasFlag(FLAG_IS_MEME))
-        wrapper->_this->defPoint->insertAfter(new CallExpr(PRIM_MOVE, wrapper->_this, wrapper_formal)); // unexecuted none/gasnet on 4/25/08
+      if (formal->hasFlag(FLAG_IS_MEME)) {
+        if (wrapper->_this != NULL) {
```

Comment:
I was really talking about the way it is used in constructors. I'd prefer it always be an ArgSymbol, rather than in some bizarre special case it being a VarSymbol.